### ISSUE TITLE
Unable to load Quick Fix Data Dictionary

### ISSUE THREAD

**Issue by Katur7** [GaryHughes/FixMaster#34]
Hi,

Thank you for a very useful extension.

I'm having some problems adding customs fields with the Quick Fix Data Dictionary Path setting.
I set the path to /home/katur/code/quickfix-custom-fields.xml.

In that file a copied this fix spec, https://raw.githubusercontent.com/quickfix/quickfix/master/spec/FIX44.xml
I also tried https://github.com/GaryHughes/FixMaster/blob/main/orchestrations/fix_repository_4_2.xml

But neither works. I get a loading bar that says: "Loading the QuickFix data dictionary..." but it never finishes loading.

I wanted to add a few custom fields so their names would show up after parsing. An example of a field would be 9882 => LiquidityFlag (char)

Do you have any example xml file for providing a couple of new fields? Or see what I am doing wrong?

**Comment by GaryHughes** [comment]
I've reproduced this and I'm investigating, thanks for the bug report!

**Comment by GaryHughes** [comment]
I've fixed this and published version 1.7.0 to the marketplace. Once again thanks for the feedback and bug report.

**Comment by Katur7** [comment]
Thank you for the quick fix.
I tried it and it is working in the new 1.7.0 version.

For anyone interested here is the minimal spec:
```
<fix type='FIX' major='4' minor='4' servicepack='0'>
  <header>
 </header>
 <messages>
  <message name='Placeholder' msgtype='0' msgcat='admin'>
  </message>
 </messages>
 <trailer>
 </trailer>
 <components>
  <component name='Placeholder'>
  </component>
 </components>
 <fields>
  <field number='8' name='BeginString' type='STRING' />
  <field number='9882' name='LiquidityFlag' type='CHAR' />
 </fields>
</fix>
```
I could not get it to load if I removed any more top level blocks
